components/credentials/src/test/java/org/cloudfoundry/credhub/services/DefaultCredentialVersionDataServiceTest.java

```diff
@@ -272,6 +277,7 @@ public void delete_onAnExistingCredential_returnsTrue() {
 
   @Test
   public void delete_onACredentialName_deletesAllCredentialsWithTheName() {
+    long nEncryptedValuesPre = encryptedValueRepository.count();
```

Comment:
Could this variable have a better name, please?

Reply:
nSomething is a common variable name style for number of something. I like it better than somethingCount, countOfSomething, or numberOfSomething.